README.md

```diff
@@ -54,7 +54,12 @@ File `package.json`:
     "scriptPreprocessor": "<rootDir>/__tests__/preprocessor.js",
   },
   "jest-webpack-alias": {
-    "webpackProfile": "dev"
+    // Required. If non-null value provided, will expect to find your webpack
+    // config stored at this key. Set to null for a regular webpack config file.
```

Comment:
Should probably mention the multi-configuration example at https://github.com/webpack/webpack/tree/master/examples/multi-compiler .  I guess they don't call them "profiles", though it's kinda vague about what they should be called.  Maybe the config option should just be named "env".